src/cmd/turnkey/pkg/encrypt.go

```diff
@@ -0,0 +1,95 @@
+package pkg
+
+import (
+	"encoding/hex"
+	"encoding/json"
+
+	"github.com/rotisserie/eris"
+	"github.com/spf13/cobra"
+
+	"github.com/tkhq/go-sdk/pkg/enclave_encrypt"
+)
+
+var (
+	// user is the user ID to import wallets and private keys with.
+	user string
+
+	// Filepath to write the import bundle to.
+	importBundlePath string
+
+	// Filepath to read the encrypted bundle from.
+	encryptedBundlePath string
+
+	// Filepath to read the plaintext from that will be encrypted.
+	plaintextPath string
+)
+
+func init() {
+	encryptCmd.Flags().StringVar(&importBundlePath, "import-bundle-path", "/import_bundle.txt", "filepath to write the import bundle to.")
+	encryptCmd.Flags().StringVar(&encryptedBundlePath, "encrypted-bundle-path", "/encrypted_bundle.txt", "filepath to read the encrypted bundle from.")
```

Comment:
I wonder if we should consider marking these as `input` or `output` to make it clear how these files are used. This'll might be more intuitive since we pipe the outputs of one command into the next during the import process.

```
--import-bundle-input <filepath>
--encrypted-bundle-output <filepath>
```

I think we can drop the `path` suffix. WDYT?

Reply:
love this idea, just updated